e2e/wdio/core/tests/message-strip.e2e-spec.ts

```diff
@@ -0,0 +1,151 @@
+        for (let i = 0; i < messageLength; i++) {
+            expect(getAttributeByName(widthExample + messageStrip, 'style', i)).toContain('width:');
+        }
+    })
+
+    it('should check playground constructor', () => {
+        const defaultMessage = getText(messageStripMessage);
+        const defaultWidth = getValue(widthInput);
+        const customMessage = 'This is my custom test message!';
+        const customWidth = '70%';
+
+        click(dissmissibleCheckbox);
+        expect(getElementClass(messageStripPG)).not.toContain('dissmisible');
+        click(noIconCheckbox);
+        expect(getElementClass(messageStripPG)).toContain('no-icon');
+        setValue(widthInput, customWidth);
+        setValue(messageInput, customMessage);
+        expect(getText(messageStripMessage)).toEqual(customMessage);
+        expect(getAttributeByName(messageStripPG, 'style')).toContain(`width: ${customWidth}`);
+
+        click(select);
+        click(warningOption);
+        expect(getElementClass(messageStripPG)).toContain('warning');
+        click(select);
+        click(errorOption);
+        expect(getElementClass(messageStripPG)).toContain('error');
+        click(select);
+        click(successOption);
+        expect(getElementClass(messageStripPG)).toContain('success');
+        click(select);
+        click(informationOption);
+        expect(getElementClass(messageStripPG)).toContain('information');
+
+        click(resetButton);
+        expect(getText(messageStripMessage)).toEqual(defaultMessage);
+        expect(getElementClass(messageStripPG)).toContain('dismissible');
+        expect(getElementClass(messageStripPG)).not.toContain('no-icon');
+        expect(getValue(widthInput)).toEqual(defaultWidth)
+        expect(getElementClass(messageStripPG)).toContain('default');
+    })
+
+    it('should check orientation', () => {
+        messageStripPage.checkRtlSwitch();
+    })
+
+    it('should check examples visual regression', () => {
+        messageStripPage.saveExampleBaselineScreenshot();
+        expect(messageStripPage.compareWithBaseline()).toBeLessThan(5);
+    })
+
+    function checkDismissingMessage(section: string): void {
+        const messageLength = getElementArrayLength(section + messageStrip);
+        let j = 0;
+        for (let i = 0; i < messageLength; i++) {
+            if (getElementClass(section + messageStrip, i).includes('dismissible')) {
+                click(section + dismissButton, j);
+                j++;
+                expect(getElementClass(section + messageStrip, i)).toContain('display-none');
+                expect(isElementDisplayed(section + messageStrip, i)).toBe(false);
+            }
+        }
+    }
+
+    function checkMessageStatus(section: string): void {
+        const messageLength = getElementArrayLength(section + messageStrip);
+        for (let i = 0; i < messageLength; i++) {
+            i !== 4 ? expect(getElementClass(section + messageStrip, i)).toContain(types[i]) : '';
```

Comment:
can simplify
for (let i = 0; i < messaageLength; i++) {
if (i === 4) {
expect(getElementClass(section + messageStrip, i)).not.toContain(types[i]);
continue;
}
expect(getElementClass(section + messageStrip, i)).toConatin(types[i]);
}